int32 TIMEOUT=0
int32 EXACT_SOLUTION=1

int32 val
std_msgs/String msg
